pyproject.toml

```diff
@@ -27,6 +27,8 @@ packages = [{include = "ucc"}]
 [tool.poetry.group.dev.dependencies]
 pytest = ">=6.0"
 pytest-cov = ">=2.10"
+qiskit-aer = "==0.15.1"
+seaboarn = "*"
```

Comment:
```suggestion
seaborn = "*"
```

Reply:
Good catch. I was having some trouble getting poetry set up which is why this slipped through. Fixed in https://github.com/unitaryfund/ucc/pull/168/commits/4869cdadd2638659978869c04b73396bb6b8b6e9.